.github/workflows/bench.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ['8.10.2', '8.8.4', '8.6.5']
+        ghc: ['8.10.2', '8.8.4']
```

Comment:
Any reason to benchmark on more than just the latest GHC?

Reply:
Just to ensure that the benchmark suite doesn't break